src/components/AvatarCropModal/AvatarCropModal.tsx

```diff
@@ -119,10 +119,17 @@ function AvatarCropModal({imageUri = '', imageName = '', imageType = '', onClose
         if (!imageUri) {
             return;
         }
-        ImageSize.getSize(imageUri).then(({width, height}) => {
-            // We need to have image sizes in shared values to properly calculate position/size/animation
-            originalImageHeight.value = height;
-            originalImageWidth.value = width;
+        // We need to have image sizes in shared values to properly calculate position/size/animation
+        ImageSize.getSize(imageUri).then(({width, height, rotation: orginalRotation}) => {
+            // On Android devices ImageSize library returns also rotation parameter.
+            if (orginalRotation === 90 || orginalRotation === 270) {
```

Comment:
Should we place the numbers in a `CONST.ts`?

Reply:
I don't think we'll use those values in any other place. It's also clear what do those numbers mean so giving them name won't improve anything.

Reply:
I agree as well. I believe the numbers are quite clear in this context. Let's keep these numbers as they are.